alf/utils/losses.py

```diff
@@ -13,34 +13,31 @@
 # limitations under the License.
 """Various function/classes related to loss computation."""
 
-import tensorflow as tf
 import gin
-from tf_agents.utils import common as tfa_common
+import torch
 
 
 @gin.configurable
 def element_wise_huber_loss(x, y):
     """Elementwise Huber loss.
 
     Args:
-        x (tf.Tensor): label
-        y (tf.Tensor): prediction
+        x (Tensor): label
+        y (Tensor): prediction
     Returns:
-        loss (tf.Tensor)
+        loss (Tensor)
     """
-    return tf.compat.v1.losses.huber_loss(
-        x, y, reduction=tf.compat.v1.losses.Reduction.NONE)
+    return torch.nn.L1Loss(reduction=None)
```

Comment:
huber loss is not L1Loss. It's SmoothL1Loss

Reply:
Changed.